syncstorage/src/db/spanner/models.rs

```diff
@@ -2128,13 +2104,11 @@ impl<'a> Db<'a> for SpannerDb {
         }
     }
 
-    #[cfg(test)]
     fn create_collection(&self, name: String) -> DbFuture<'_, i32> {
```

Comment:
Yeah I think it might be wise to do something with them. Putting them into a separate trait would at least make them slightly harder to use by mistake 🤔 I'll log an issue for this